SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/UtilsShared/DataBrokerProtectionSettings.swift

```diff
@@ -172,6 +172,12 @@ extension DataBrokerProtectionSettings {
         }
     }
 
+    @discardableResult
+    public static func rotateDeviceIdentifier() -> String {
+        defaults.removeObject(forKey: deviceIdentifierKey)
+        return deviceIdentifier
```

Comment:
I think it would be good to explain the side effect of using the getter here.
Although it doesn't really matter since this should definitely be temporary

Reply:
Updated this to instead add a count to rotated IDs, so we don't have the same getter shenanigans happening any more.